core/server/common/src/main/java/alluxio/master/journal/raft/RaftJournalSystem.java

```diff
@@ -650,8 +643,8 @@ public synchronized Map<ServiceType, GrpcService> getJournalServices() {
   private void catchUp(JournalStateMachine stateMachine, RaftJournalAppender client)
       throws TimeoutException, InterruptedException {
     long startTime = System.currentTimeMillis();
-    long waitBeforeRetry = ServerConfiguration.global()
-        .getMs(PropertyKey.MASTER_EMBEDDED_JOURNAL_CATCHUP_RETRY_WAIT);
```

Comment:
What's a bit confusing to me is that some configuration properties come from mConf (RaftJournalConfiguration) which is set using property keys and some just get the property key directly. I wonder if there is a reason for this and if it is not better to just use one or the other?

Reply:
Yea I think the only reason to use the config class would be if there were multiple instance of RaftJournalSystem that might use different configurations, but I don't think this will ever be the case? I think now best practice is to always get values directly from the property keys instead of caching them in some local config.

Reply:
Removed the `RaftJournalConfiguration` and replaced some with `private final` variables and others by the actual `PropertyKey` they corresponded to. I also moved the input validation to the constructor and only log an error instead of throwing an exception.